src/views/AuthorityView/AuthorityView.js

```diff
@@ -30,6 +30,9 @@ import {
   CalloutContext,
 } from '@folio/stripes/core';
 import MarcView from '@folio/quick-marc/src/QuickMarcView/QuickMarcView';
+
+import { useAuthorityMappingRules } from '@folio/stripes-authority-components/lib/queries';
```

Comment:
you can just import from `@folio/stripes-authority-components` because all content of `/lib` is exported automatically and the import will look cleaner

Reply:
Good point, thanks